q2_sample_classifier/plugin_setup.py

```diff
@@ -454,6 +456,8 @@
     parameter_descriptions={
         'truth': 'Metadata column (true values) to plot on y axis.',
         'missing_samples': parameter_descriptions['base']['missing_samples'],
+        'vmin': 'The minimum color value for the heatmap',
```

Comment:
How about we modify from the seaborn descriptions:
'vmin': 'The minimum value to use for anchoring the colormap. If "auto", vmin is set to the minimum value in the data.', 
'vmax': 'The maximum value to use for anchoring the colormap. If "auto", vmax is set to the maximum value in the data.'

Reply:
@Oddant1 please see this old change needed